packages/backend-modules/republik-crowdfundings/graphql/resolvers/PackageOption.js

```diff
@@ -20,6 +20,12 @@ module.exports = {
 
     return autoPay
   },
+  fixedPrice({ fixedPrice }) {
```

Comment:
Without looking in detail over the whole PR, just a quick comment here:
Not necessary to fetch these two props here. They are already included in the parent data and do not have to be returned again (it's called overfetching).